src/main/core/main.c

```diff
@@ -250,6 +259,19 @@ gint main_runShadow(gint argc, gchar* argv[]) {
         }
     }
 
+    if (_setSchedFifo) {
+        struct sched_param param = {0};
+        param.sched_priority = 1;
+        int rc = sched_setscheduler(0, SCHED_FIFO, &param);
+
+        if (rc != 0) {
+            fprintf(stderr, "Could not set SCHED_FIFO!\n");
```

Comment:
The logger has already been set up at this point, so can we log this with our logger? How about:
```
error("Could not set real-time scheduler mode to SCHED_FIFO!")
```

Reply:
Done